etc/profile-m-z/remmina.profile

```diff
@@ -12,6 +12,8 @@ noblacklist ${HOME}/.local/share/remmina
 
 # Allow ssh (blacklisted by disable-common.inc)
 include allow-ssh.inc
+# Remmina needs to load some modules
```

Comment:
Please follow our commenting format.

Suggestion:
```
# Allow python (blacklisted by disable-interpreters.inc)
#include allow-python2.inc
```

Reply:
What exactly do you want me to do here? To add `#include allow-python2.inc` or to add the "blacklisted by ..." comment?

Reply:
```suggestion

# Allow python (blacklisted by disable-interpreters.inc)
```